source/agent/addons/quic/WebTransportFrameDestination.cc

```diff
@@ -45,4 +54,83 @@ NAN_METHOD(WebTransportFrameDestination::newInstance)
     WebTransportFrameDestination* obj = new WebTransportFrameDestination();
     obj->Wrap(info.This());
     info.GetReturnValue().Set(info.This());
+}
+
+NAN_METHOD(WebTransportFrameDestination::addDatagramOutput)
+{
+    WebTransportFrameDestination* obj = Nan::ObjectWrap::Unwrap<WebTransportFrameDestination>(info.Holder());
+    if (obj->m_datagramOutput) {
```

Comment:
Should you call `obj->m_datagramOutput->FrameDestination()->setDataSource(nullptr);` for previous datagramOutput here?

Reply:
Done.